dd-sdk-android/src/main/kotlin/com/datadog/android/log/Logger.kt

```diff
@@ -169,6 +168,30 @@ internal constructor(internal var handler: LogHandler) {
         internalLog(priority, message, throwable, attributes)
     }
 
+    /**
+     * Sends a log message with strings for error information.
+     *
+     * @param priority the priority level (must be one of the Android Log.* constants)
+     * @param message the message to be logged
+     * @param errorKind the kind of error to be logged with the message
+     * @param errorMessage the message from the error to be logged with this message
+     * @param errorStack the stack trace from the error to be logged with this message
+     * @param attributes a map of attributes to include only for this message. If an attribute with
+     * the same key already exist in this logger, it will be overridden (just for this message)
+     */
+    @JvmOverloads
+    @Suppress("LongParameterList")
+    fun log(
+        priority: Int,
+        message: String,
+        errorKind: String?,
+        errorMessage: String?,
+        errorStack: String?,
```

Comment:
minor suggestion to have the consistency with the similar method in `RumMonitor`

```suggestion
        errorStacktrace: String?,
```